edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Docker/EdgeAgentDockerModule.cs

```diff
@@ -21,9 +23,24 @@ public override bool Equals(IModule<DockerConfig> other)
                 return false;
             if (ReferenceEquals(this, other))
                 return true;
-            return string.Equals(this.Name, other.Name) &&
-                string.Equals(this.Type, other.Type) &&
-                string.Equals(this.Config.Image, other.Config.Image);
+
+            if (!string.Equals(this.Name, other.Name) ||
+                !string.Equals(this.Type, other.Type) ||
```

Comment:
I think we previously missed "Version" property.

Reply:
I think that was intentional. @varunpuranik?

Reply:
Yup, we don't want to compare version. It is not used.